Stratis.Bitcoin.Features.LightWallet/LightWalletSyncManager.cs

```diff
@@ -197,7 +241,7 @@ public void SyncFromDate(DateTime date)
             // before we start syncing we need to make sure that the chain is at a certain level.
             // if the chain is behind the date from which we want to sync, we wait for it to catch up, and then we start syncing.
             // if the chain is already past the date we want to sync from, we don't wait, even though the chain might not be fully downloaded.
-            if (this.chain.Tip.Header.BlockTime.LocalDateTime < date)
+            if (this.chain.Tip != null && this.chain.Tip.Header.BlockTime.LocalDateTime < date)
```

Comment:
Can the tip be null? what shall we do if that's the case? @dangershony

Reply:
The tip can not be null, it will default to genies (height zero)